src/DeviceInput/InputDevices/webDeviceInputSystem.ts

```diff
@@ -41,7 +45,7 @@ export class WebDeviceInputSystemImpl implements IDeviceInputSystem {
     private _pointerActive: boolean = false;
     private _elementToAttachTo: HTMLElement;
     private _engine: Engine;

```

Comment:
nit: could be readonly. maybe can do a pass on this later.

Reply:
Easy enough to change

Reply:
There are more that can be readonly, but no need to do this now.